docs/config.md

```diff
@@ -130,7 +130,10 @@ Each router must be configured as an object with the following params:
   (default is protocol dependent)
 * *failFast* -- If `true`, connection failures are punished more aggressively.
   Should not be used with small destination pools. (default: false)
-* *timeoutMs* -- Per-request timeout in milliseconds. (default: no timeout)
+* *timeoutMs* -- Per-request timeout in milliseconds. (default: no  timeout)
```

Comment:
Smallest of nits -- you're introducing an extra space between "no" and "timeout".